R/exec_profile.R

```diff
@@ -4,7 +4,7 @@ exec_profile <- function(.fn, indicator, indicator_after) {
     product_raw <- unnest_product(tilt_indicator_output) |>
       extend_with_columns_from_arguments_of_tilt_indicator(indicator, .fn)
 
-    co2_range <- create_co2_range(product_raw)
+    co2_range <- create_co2_range(product_raw, amount = 0.5)
```

Comment:
How would a user (e.g. you, me, Anne) modify `amount` when we call the corresponding `profile*()` function?

As far as I can tell there is no way to change `0.5` other than submitting a PR.

There are two ways to allow that:

1. Hard-core: Add `amount` in the corresponding `profile*()` function so that a user can pass it through it.
2. Ligher: Pass it via an option, i.e. replace `amount = 0.5` with `amount = getOption("tiltIndicatorAfter.jitter_amount", default = 0.5)`. For example:


``` r
set.seed(1)

f <- function(data) {
  # more code
  tiltIndicator:::jitter_range(data, amount = getOption("tiltIndicatorAfter.jitter_amount", default = 0.5))
}

data <- tibble::tibble(min = 1:3, max = 1:3)

options(tiltIndicatorAfter.jitter_amount = 0.1)
f(data)
#> # A tibble: 3 × 4
#>     min   max min_jitter max_jitter
#>   <int> <int>      <dbl>      <dbl>
#> 1     1     1      0.911       1.09
#> 2     2     2      1.94        2.12
#> 3     3     3      2.92        3.19

options(tiltIndicatorAfter.jitter_amount = 100)
f(data)
#> # A tibble: 3 × 4
#>     min   max min_jitter max_jitter
#>   <int> <int>      <dbl>      <dbl>
#> 1     1     1      -36.4       1.46
#> 2     2     2      -36.4      89.0 
#> 3     3     3     -159.      298.
```


--

I would start with the lighter approach, since the usage of this argument is mostly for internal users (you, me, Anne, etc.). 

The hard-core approach is not a bad idea but maybe something to think a bit more:

* It modifies the signature of some but not all of the `profile*()` functions, meaning it introduces an inconsistency across the set of `profile*()` functions.
* It requires a bit more work to maintain -- e.g. it'll drive a similar change in tiltWorkflows.

In any case, it'll be important to document it's usage, at least in an example in the help file of the corresponding `profile*()` function.